src/test/librbd/managed_lock/test_mock_ReleaseRequest.cc

```diff
@@ -17,7 +19,36 @@ struct Traits<MockImageCtx> {
   typedef librbd::MockImageWatcher Watcher;
 };
 }
-}
+
+namespace managed_lock {
+
+template <>
+struct BreakLockRequest<librbd::MockImageCtx> {
```

Comment:
I'll have to take a closer look later -- definitely don't want to have to include CC files outside their own unit test.